### PR TITLE
Moved keptn brigde to "Bridge' section on same level as references

### PR DESCRIPTION
**Pull request by Ciggzy1312**
Signed-off-by: Deepayan <deepayanmukherjee1312@gmail.com>

-> Moved Keptn bridge under "Bridge" subsection
-> Bridge subsection is kept on same level as Quality Gates and References as per docs proposal

-> The description for the Bridge section is yet to be decided
-> Implementation of "Bridge" entry to Glossary and Concepts article are yet to be discussed

### CODE REVIEW

**Review by StackScribe — COMMENTED** [review]
Good idea to just get the Bridge files out of the Reference section for now.  But I think we want to put them in a section on the same level as Quality Gates, so within the release-specific section.  That will retain the general shape of the current doc set until we revamp the whole thing.

I think it is okay to pull the Reference section out of the release-specific section, so it will be at the same level as Concepts.  That is just a little tweak in the structure  ;-). Other opinions?

File: content/docs/0.13.x/Bridge/_index.md [hunk 1]
```diff
@@ -0,0 +1,3 @@
+---
+title: Bridge
```
Comment (oleg-nenashev) [inline content/docs/0.13.x/Bridge/_index.md:2]:
```suggestion
title: Bridge (Web UI)
```

**Review by oleg-nenashev — APPROVED** [review]
No objections from me. Maybe it makes sense to add a second term for Bridge which is more friendly to newcomer users

**Comment by StackScribe** [comment]
I just found https://keptn.sh/docs/concepts/architecture/#keptn-bridge -- needs some work sometime but it does link to the Bridge pages that are currently in the Reference section.  Those links should be replaced with links to this new section in this PR.

**Comment by johannes-b** [comment]
Hi @Ciggzy1312, 

thanks for the PR. Please take into account that there is already a folder for the upcoming `0.14.x` release. Hence I would like to ask you to perform the same changes in `0.14.x` (https://github.com/keptn/keptn.github.io/tree/master/content/docs/0.14.x) as well to not lose them in the future. Thanks!

FYI: @oleg-nenashev & @StackScribe

**Pull request merged**